lib/mumuki/laboratory.rb

```diff
@@ -9,23 +9,41 @@ module Laboratory
   end
 end
 
-require 'mumukit/assistant'
-require 'mumukit/randomizer'
-require 'mumukit/inspection'
-require 'mumukit/bridge'
-require 'mumukit/content_type'
-require 'mumukit/directives'
+require 'mumuki/domain'
+require 'mumukit/login'
+require 'mumukit/nuntius'
+require 'mumukit/platform'
+
 require 'kaminari'
 require 'bootstrap-kaminari-views'
 
-require_relative './laboratory/mumukit/auth'
-require_relative './laboratory/mumukit/login'
-require_relative './laboratory/mumukit/nuntius'
-require_relative './laboratory/mumukit/platform'
+Mumukit::Nuntius.configure do |config|
+  config.app_name = 'laboratory'
+end
+
+Mumukit::Platform.configure do |config|
+  config.application = Mumukit::Platform.laboratory
+  config.web_framework = Mumukit::Platform::WebFramework::Rails
+end
+
+class Mumuki::Laboratory::Engine < ::Rails::Engine
+  config.i18n.available_locales = Mumukit::Platform::Locale.supported
+end
+
+module Mumukit::Platform::OrganizationMapping::Path
```

Comment:
I think this block is no longer necessary :thinking:

Reply:
uhhhhh seems like it was necessary after all! :sweat_smile: removing it broke login in this branch, so we should restore it before merging :warning:

Reply:
Yeah, but i'd like to see hyper being called here :tada: